### PR TITLE
Adds README

### LINKED ISSUE #10: Add README with run instructions

### PR DESCRIPTION
**Pull request by NicEastvillage**
This readme describes the aim of the application and gives an example use case.

Watch out of spelling mistakes.

Resolves #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# atl-checker
+
+This application can do model checking of alternating-time temporal logic (ATL) on concurrent game structures (CGS).
```
Comment (cogitantium) [inline README.md:3]:
```suggestion
This application can do model checking of alternating-time temporal logic (ATL) on concurrent game structures (CGSs).
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+It uses a on-the-fly evaluation method as opposed to calculating the fixed point like PRISM do.
+This lazy method avoids having to generate the whole graph, and is therefore faster in most cases (has yet to be proved).
+
+The atl-checker also uses a custom language to describe CGSs in a lazy manner. We call it LCGS and
```
Comment (cogitantium) [inline README.md:7]:
```suggestion
The ATL-checker also uses a custom language to describe CGSs in a lazy manner. We call it LCGS and
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Here `"players": [0]` refers to Billy, and `"proposition": 0` refers to Billy's label called "alive".
+
+We now call the atl-checker with the following arguments
+
+```
+./atl-checker solver -m standoff.lcgs -f billy-stays-alive.json
+```
+
+The result turns out to be FALSE. Billy canot guarantee to stay alive.
```
Comment (cogitantium) [inline README.md:62]:
```suggestion
The result turns out to be FALSE. Billy cannot guarantee to stay alive.
```

**Review by cogitantium — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+If a cowboy is hit by two bullets, he dies.
+
+We describe this scenario using LCGS as seen below
+
+```
+const max_health = 2;
+
+player billy = cowboy [target1=p2, target2=p3];
+player clayton = cowboy [target1=p3, target2=p1];
+player jesse = cowboy [target1=p1, target2=p3];
```
Comment (GeneralDenmark) [inline README.md:23]:
```suggestion
player billy = cowboy [target1=clayton, target2=jesse];
player clayton = cowboy [target1=jesse, target2=billy];
player jesse = cowboy [target1=billy, target2=clayton];
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    [wait] true;
+    [shoot_right] health > 0 & target1.health > 0;
+    [shoot_left] health > 0 & target2.health > 0;
+
+endtemplate
+```
+
+We can now use ATL logic and query whether Billy has a strategy to guarantees that he survives.
+The ATL formula is givin in json:
+
```
Comment (cogitantium) [inline README.md:41]:
```suggestion
Using labels in propositions require using their indices. These are zero-indexed, receiving an index in the order they're defined starting with globals, then templates for all players, starting with the first defined player. 
```

**Review by cogitantium — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (cogitantium) [reply README.md:41]: Skal måske lige rykkes op en tak

Comment (NicEastvillage) [reply README.md:41]: I think that is too much detail for the example.

**Review by cogitantium — APPROVED** [review]

**Review by GeneralDenmark — APPROVED** [review]
niCE

**Pull request merged**